universal-application-tool-0.0.1/conf/routes

```diff
@@ -21,6 +21,13 @@ GET     /admin/programs/:id/edit    controllers.admin.AdminProgramController.edi
 POST    /admin/programs/:id         controllers.admin.AdminProgramController.update(request: Request, id: Long)
 POST    /admin/programs             controllers.admin.AdminProgramController.create(request: Request)
 
+# A controller for a page for an admin to view, edit, and create questions
+GET     /admin/questions             controllers.admin.QuestionController.list(renderAs: String ?= "table")
```

Comment:
controller method should be `index`.

https://github.com/seattle-uat/universal-application-tool#html-routing-convention

Reply:
Done.